official/benchmark/models/resnet_imagenet_main.py

```diff
@@ -275,16 +365,23 @@ def run(flags_obj):
   if not strategy and flags_obj.explicit_gpu_placement:
     no_dist_strat_device.__exit__()
 
+  if flags_obj.save_files_to:
```

Comment:
hmm, I can't see where it is ensured that we are in clustering mode at this point

Reply:
yes, that's a good point and I fixed in the latest commit.

Reply:
Also, more tests were added in the latest commit ...

Reply:
yes, sorry, I totally overlooked that

Reply:
no worries.